src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -450,7 +450,7 @@ else if (building.getTask().equals(BuildingGuardTower.Task.FOLLOW))
                 final EntityPlayer player = building.getPlayer();
                 if (player != null)
                 {
-                    LanguageHandler.sendPlayerMessage(building.getPlayer(), LanguageHandler.format("com.minecolonies.coremod.job.guard.switch"));
+                    LanguageHandler.sendPlayerMessage(building.getPlayer(), "com.minecolonies.coremod.job.guard.switch");
```

Comment:
Sorry, I didn't realize sendPlayerMessage had been changed, we should probably deprecate sendPlayerLocalizedMessage as it just called sendPlayerMessage now.